src/utils/event-manager/constants.js

```diff
@@ -33,7 +33,6 @@ export const BASIC_EVENT_ALIASES = {
  * this block maps event names to the Recognizers required to detect the events.
  */
 export const EVENT_RECOGNIZER_MAP = {
-  click: 'tap',
```

Comment:
I changed the recognizer look up to use the aliased event name (click -> tap) so this entry is no longer necessary.